lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -2700,7 +2700,7 @@ DefPrimitive('\@addtoreset{}{}', sub {
     return; });
 
 DefMacro('\value{}', sub {
-    ExplodeText(CounterValue(ToString(Expand($_[1])))->valueOf); });
+    T_CS("\\c@".ToString(Expand($_[1]))) });
```

Comment:
A side-remark here in passing - I've seen a number of errors due to "No more Input" that have to do with the Expand technique of using a standalone Mouth for the expansion. Some of the low-level tricks in the sty files end up reaching beyond the argument, so Expand may need an upgrade that can grab the next Mouth from the gullet when its sandbox is not enough?

Reply:
Not something to address now, I will eventually have a perfect example that shows this, which we can use to upgrade. But bookkeeping to remember.